### PR TITLE
Ensure content of META-INF/services is merged correctly for fatjar

### PR DESCRIPTION
**Pull request by bjorncs**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: vespa-feed-client-cli/src/maven/jar-with-dependencies.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    <includeBaseDirectory>false</includeBaseDirectory>
+    <dependencySets>
+        <dependencySet>
+            <outputDirectory>/</outputDirectory>
+            <useProjectArtifact>true</useProjectArtifact>
+            <unpack>true</unpack>
+            <scope>runtime</scope>
+        </dependencySet>
+    </dependencySets>
+</assembly>
```
Comment (baldersheim) [inline vespa-feed-client-cli/src/maven/jar-with-dependencies.xml:27]: Missing eol at eof

File: vespa-feed-client-cli/src/maven/create-zip.xml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root. -->
-<assembly xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2"
+<assembly xmlns="http://maven.apache.org/ASSEMBLY/2.1.1"
           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-          xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2 http://maven.apache.org/xsd/assembly-1.1.2.xsd">
+          xsi:schemaLocation="http://maven.apache.org/ASSEMBLY/2.2.1 https://maven.apache.org/xsd/assembly-2.1.1.xsd">
```
Comment (baldersheim) [inline vespa-feed-client-cli/src/maven/create-zip.xml:5]: 1.1.2 => 2.1.1
Intentional, or accidentally reversed

**Review by baldersheim — DISMISSED** [review]
2 comments to consider.

**Review by gjoranv — COMMENTED** [review]
👍 Why not use shade-plugin?

**Comment by bjorncs** [comment]
@gjoranv I don't remember the rationale of using assembly over shade plugin. It could be due to the zip goal.

**Pull request merged**